tests/unit/modules/test_silence_info.py

```diff
@@ -35,7 +35,8 @@ def test_get_single_silence(self, mocker):
             silence_info.main()
 
         _client, path = get_mock.call_args[0]
-        assert path == "/silenced/subscription:*"
+        print(path)
```

Comment:
Is this a leftover from debugging ;)?

Reply:
I never debug, my code works on the first try ;) Nice catch indeed. Thanks @aljazkosir.